docs/development/releasing.rst

```diff
@@ -89,6 +84,16 @@ packages that use the full bugfix/maintenance branch approach.)
 
       $ git checkout v1.2.x
 
+#. Make sure that the continuous integration services (e.g., Travis) are passing
```

Comment:
Perhaps
```suggestion
#. Make sure that the continuous integration services (e.g., CircleCI) are passing
```
just because of all the uncertainty in Travis' future?

Reply:
What about listing multiple? I regularly point to this docs for folks looking for instructions, and in the template we provide more for travis than for circleCI, thus many projects only use that one.

Reply:
OK, I added both.